connector/src/main/java/org/geysermc/connector/bootstrap/GeyserBootstrap.java

```diff
@@ -120,4 +119,8 @@ default String getMinecraftServerVersion() {
     default SocketAddress getSocketAddress() {
         return null;
     }
+
+    default String logsPath() {
+        return "logs/latest.log";
```

Comment:
I would make this return a Path instead of String

Also, since the default is "logs/latest.log", you don't need to override it on platforms that have their latest log in that location. You could mention in the javadoc that the default provides a Path of "logs/latest.log"